### ISSUE TITLE
Accessibility testing for site

### ISSUE THREAD

**Issue by c-fuchs** [hackforla/shared-housing#175]
### Overview
An estimated 45% of the homeless are disabled, so we need to be sure our site is accessible.

There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  While the legal guidelines can be found [here](https://www.ada.gov/pcatoolkit/chap5toolkit.htm), for guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/) and another [list](https:%3A%2F%2Fwww.mediacurrent.com%2Fblog%2F5-website-accessibility-checkers)

**Comment by abregorivas** [comment]
Please note: If you are using material-ui as your component library a lot of the accessibility is handled for you as long as you add the proper prop values where appropriate and you use proper hierarchy for headings and proper semantic tags.  Therefore, using wave and then lighthouse would provide a AA rating. Just my two cents!!!
